src/app/components/sdrf-recommend-panel/sdrf-recommend-panel.component.ts

```
@@ -0,0 +1,2827 @@
+    return recs;
+  });
+
+  readonly sortedRecommendations = computed(() => {
+    const recs = [...this.filteredRecommendations()];
+    const sort = this.sortBy();
+
+    recs.sort((a, b) => {
+      switch (sort) {
+        case 'confidence':
+          const confOrder = { high: 0, medium: 1, low: 2 };
+          return confOrder[a.confidence] - confOrder[b.confidence];
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)